_rules/80f0bf.md

```diff
@@ -0,0 +1,40 @@
+---
+name: No auto-play audio
+test_type: composite
+
+description: |
+  This rule checks that there is either no auto-play audio or that there is a mechanism to control it.
```

Comment:
This rule doesn't check that the audio/video has auto-play. There seems to be an underlying assumption that the page is tested before any interaction has happened to it. If you run this rule after clicking a play button, this rule doesn't work. That assumption applies to all these rules.